hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/tenant/OMTenantCreateRequest.java

```diff
@@ -292,27 +291,27 @@ public OMClientResponse validateAndUpdateCache(
       // Create tenant
       // Add to tenantStateTable. Redundant assignment for clarity
       final String bucketNamespaceName = volumeName;
-      final List<String> policyNamesList = new ArrayList<>();
       // Populate policy ID list
-      // TODO: Check if both policies are actually used. Remove if not
-      policyNamesList.add(tenantDefaultPolicies);
-      final String bucketPolicyGroupName =
-          tenantId + OzoneConsts.DEFAULT_TENANT_BUCKET_POLICY_SUFFIX;
+      final String bucketNamespacePolicyName =
+          OMMultiTenantManager.getDefaultBucketNamespacePolicyName(tenantId);
       final String bucketPolicyName =
-          bucketPolicyGroupName + OzoneConsts.DEFAULT_TENANT_POLICY_NAME_SUFFIX;
-      policyNamesList.add(bucketPolicyName);
-
-      final OmDBTenantState omDBTenantInfo = new OmDBTenantState(
-          tenantId, bucketNamespaceName, policyNamesList);
+          OMMultiTenantManager.getDefaultBucketNamespacePolicyName(tenantId);
```

Comment:
Should be `OMMultiTenantManager.getDefaultBucketPolicyName(tenantId);`?

Reply:
Good catch